CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/clean/SplitNodesPlugin.java

```diff
@@ -91,7 +91,7 @@ public PluginParameters createParameters() {
         final PluginParameter<StringParameterValue> split = StringParameterType.build(SPLIT_PARAMETER_ID);
         split.setName("Split Character(s)");
         split.setDescription("A new term will be extracted from the first instance of this character(s) in the Identifier");
-        split.setStringValue(null);
+        split.setStringValue("");
```

Comment:
My preference would be to do a null check further on rather than set this to an empty string. I say this because if via the ReST API we set it to null then we easily break the plugin. What do you think @aldebaran30701 ?

Reply:
Good point, I'm adding a check to it instead. I'll test it and push the change in a moment. 👍

Reply:
Thank you @aldebaran30701